app/docker/views/configs/create/createConfigController.js

```diff
@@ -90,4 +89,30 @@ function ($scope, $state, Notifications, ConfigService, Authentication, FormVali
   $scope.editorUpdate = function(cm) {
     $scope.formValues.ConfigContent = cm.getValue();
   };
+
+  function initView() {
+    if ($transition$.params().id === '') {
+        // display code editor component in new configs
+        $scope.formValues.displayCodeEditor = true;
+    } else {
+      ConfigService.config($transition$.params().id)
+      .then(function success(data) {
+        $scope.formValues.Name = data.Name + "_Copy";
+        $scope.formValues.Data = data.Data;
+        $scope.formValues.Labels = [];
+        for (var label in data.Labels) {
```

Comment:
I'd rather see a classic `for` loop instead of a `for in`, can you update it?

```javascript
for (var i = 0; i < array.length; i++) {
    var item = array[i];
    ...
  }
```

Reply:
I'm not sure that it is possible because `data.Labels` is an object, not an array. I'm not a javascript expert... is there a more elegant way than `for in` loop to iterate over object properties?

Reply:
You can use lodash `keys` method to retrieve the properties of the object: https://lodash.com/docs/4.17.10#keys

```javascript
  var labels = _.keys(data.Labels);
  for (var i = 0; i < labels.length; i++) {
    var labelName = labels[i];
    // var labelValue = data.Labels[labelName];
    ...
  }
```

Reply:
Done, thank you for help.